Pensel/firmware/applications/pensel_v2.c

```diff
@@ -86,108 +130,125 @@ void clear_critical_errors(void) {
  */
 int main(void)
 {
-    ret_t retval;
-    uint32_t subcount = 0;
+    uint8_t i;
```

Comment:
consider giving this variable a more descriptive name since it is not clear what it is used for until you dig into `scheduler_add`